Marlin/ultralcd.cpp

```diff
@@ -848,6 +859,39 @@ void kill_screen(const char* lcd_msg) {
 
   /**
    *
+   * "Driver current control" submenu items
+   *
+   */
+  #ifdef DAC_STEPPER_CURRENT
+  static void dac_driver_getValues() 
+  {
+    driverX = mcp4728_getDrvPct(0);
+    driverY = mcp4728_getDrvPct(1);
+    driverZ = mcp4728_getDrvPct(2);
+    driverE = mcp4728_getDrvPct(3);
+  } 
+
+  static void dac_driver_commit() { mcp4728_setDrvPct(driverX, driverY, driverZ, driverE); }
+
+  static void dac_driver_eeprom_write() { mcp4728_eepromWrite(); }
+  
+  static void lcd_dac_menu()
+  {
+   dac_driver_getValues();
+   START_MENU();    
+    MENU_ITEM(back, MSG_CONTROL);
+    MENU_ITEM_EDIT_CALLBACK(int3, "Driver X%", &driverX, 0, 100, dac_driver_commit);
```

Comment:
These will need to be made translatable strings in the language files. I suggest that you use:
```cpp
MENU_ITEM_EDIT_CALLBACK(int3, MSG_X " " MSG_DAC_PERCENT, &driverPercent[X_AXIS], 0, 100, dac_driver_commit);
```
…where the message string is defined as…
```cpp
#define MSG_DAC_PERCENT "Driver %"
```